src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -150,6 +152,42 @@ public PersistentDataStore() {
     return messages;
   }
 
+  /**
+   * Loads all Media objects from the Datastore service and returns them in a List, sorted in
+   * ascending order by creation time.
+   *
+   * @throws PersistentDataStoreException if an error was detected during the load from the
+   *     Datastore service
+   */
+  public List<Media> loadMedia() throws PersistentDataStoreException {
+
+    List<Media> media = new ArrayList<>();
+
+    // Retrieve all conversations from the datastore.
+    Query query = new Query("chat-media").addSort("creation_time", SortDirection.ASCENDING);
```

Comment:
One thing to think about is that there could be many different types of media types (e.g., jpg, png), and media purposes (profile pictures vs other usages). Then given a piece of media in the chat-media table, how do I know what it is used for?

Reply:
Could I somehow use enums in the media class for this?

Reply:
Sure, what is your implementation to support this is up to you.